### DISCUSSION TITLE
Rendering issue with checkboxes and multi-selects components

### DISCUSSION THREAD

**Discussion by alexDombrovskiClarius** [primefaces/community#1510] [PrimeVue]
Hello.

I am new to Primevue and in my project I am getting weird rendering issue with check-boxes and multi-selects. 
After checkbox or multi-select is selected checkbox-box is shrinking and second check-mark shown. 
Checkbox:
![image](https://github.com/primefaces/community/assets/118212259/034158bb-9337-4f69-9ae7-05dfdcf8d042)
Multi-select:
![image](https://github.com/primefaces/community/assets/118212259/55b9a452-6ac1-45af-ad20-c8e42ab0cc27)

Also, I noticed that `data-pc-section-box` is missing `p-checkbox-box` class. I believe that is what causing this issue, but I am not sure how to fix it. 
![image](https://github.com/primefaces/community/assets/118212259/cb1ba413-8111-4b83-9b28-40e924e54d77)
I am using Primevue `3.50.0`.
Can someone please help me with that? 

Thank you in advance.

**Comment by ipimpat** [comment]
Are you using Tailwind?

https://github.com/primefaces/primevue-tailwind/issues/88
